modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PowerShellClientCodegen.java

```diff
@@ -905,9 +921,19 @@ public String toParamName(String name) {
         for (Object _mo : models) {
             Map<String, Object> _model = (Map<String, Object>) _mo;
             CodegenModel model = (CodegenModel) _model.get("model");
+            CodegenProperty lastWritableProperty = null;
 
             for (CodegenProperty cp : model.allVars) {
                 cp.vendorExtensions.put("x-powershell-data-type", getPSDataType(cp));
+                if(this.discardReadOnly && !cp.isReadOnly) {
+                    lastWritableProperty = cp;
+                }
+            }
+
+            // Mark the last readonly false property
+            if(this.discardReadOnly && lastWritableProperty != null) {
+                lastWritableProperty.vendorExtensions.put("x-lastWritable", true);
```

Comment:
I'll rename the extension to `x-ps-last-writable` instead to make it conform to the naming convention